2020-11-02 04:47:11,748 - root - ERROR - Exception : 
Traceback (most recent call last):
  File "D:/00Code/0-old/auto_zhihu_clipper/evernote_main.py", line 54, in zhihu_process
    WebDriverWait(driver, 20, 0.5).until(lambda driver: driver.find_element_by_xpath(xpath_content))  # 寻找重排
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "D:/00Code/0-old/auto_zhihu_clipper/evernote_main.py", line 54, in <lambda>
    WebDriverWait(driver, 20, 0.5).until(lambda driver: driver.find_element_by_xpath(xpath_content))  # 寻找重排
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 978, in find_element
    'value': value})['value']
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=85.0.4183.121)
  (Driver info: chromedriver=72.0.3626.7 (efcef9a3ecda02b2132af215116a03852d08b9cb),platform=Windows NT 10.0.18363 x86_64)


2020-11-02 08:36:44,375 - root - ERROR - Exception : 
Traceback (most recent call last):
  File "D:/00Code/0-old/auto_zhihu_clipper/evernote_main.py", line 84, in zhihu_process
    Xpath_wait_click_or_input(driver, '//div[text()="整个网页"]')
  File "D:\00Code\0-old\auto_zhihu_clipper\utils\utils.py", line 27, in Xpath_wait_click_or_input
    driver.find_element_by_xpath(xpath).click()
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 978, in find_element
    'value': value})['value']
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[text()="整个网页"]"}
  (Session info: chrome=85.0.4183.121)
  (Driver info: chromedriver=72.0.3626.7 (efcef9a3ecda02b2132af215116a03852d08b9cb),platform=Windows NT 10.0.18363 x86_64)


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:/00Code/0-old/auto_zhihu_clipper/evernote_main.py", line 84, in zhihu_process
    Xpath_wait_click_or_input(driver, '//div[text()="整个网页"]')
  File "D:\00Code\0-old\auto_zhihu_clipper\utils\utils.py", line 27, in Xpath_wait_click_or_input
    driver.find_element_by_xpath(xpath).click()
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 978, in find_element
    'value': value})['value']
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[text()="整个网页"]"}
  (Session info: chrome=85.0.4183.121)
  (Driver info: chromedriver=72.0.3626.7 (efcef9a3ecda02b2132af215116a03852d08b9cb),platform=Windows NT 10.0.18363 x86_64)


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:/00Code/0-old/auto_zhihu_clipper/evernote_main.py", line 82, in zhihu_process
    driver.switch_to.frame("evernoteClipperTools")  # 切换到 嵌入iframe代码
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\switch_to.py", line 89, in frame
    self._driver.execute(Command.SWITCH_TO_FRAME, {'id': frame_reference})
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchFrameException: Message: no such frame
  (Session info: chrome=85.0.4183.121)
  (Driver info: chromedriver=72.0.3626.7 (efcef9a3ecda02b2132af215116a03852d08b9cb),platform=Windows NT 10.0.18363 x86_64)


2020-11-06 21:35:53,448 - root - ERROR - Exception : 
Traceback (most recent call last):
  File "D:/00Code/0-old/auto_zhihu_clipper/evernote_main.py", line 92, in zhihu_process
    Xpath_wait_click_or_input(driver, '//div[@class="children"]//div[contains(@class, "withoutCollapse") and @title="z知乎"]')
  File "D:\00Code\0-old\auto_zhihu_clipper\utils\utils.py", line 25, in Xpath_wait_click_or_input
    wait_to_visible(driver, xpath)
  File "D:\00Code\0-old\auto_zhihu_clipper\utils\utils.py", line 14, in wait_to_visible
    ui.WebDriverWait(driver, timeout).until(EC.visibility_of_element_located((By.XPATH, Xpath)))
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 128, in __call__
    return _element_if_visible(_find_element(driver, self.locator))
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 415, in _find_element
    raise e
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 411, in _find_element
    return driver.find_element(*by)
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 978, in find_element
    'value': value})['value']
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: target frame detached
  (Session info: chrome=85.0.4183.121)
  (Driver info: chromedriver=72.0.3626.7 (efcef9a3ecda02b2132af215116a03852d08b9cb),platform=Windows NT 10.0.18363 x86_64)


2020-11-06 21:36:32,594 - root - ERROR - Exception : 
Traceback (most recent call last):
  File "D:/00Code/0-old/auto_zhihu_clipper/evernote_main.py", line 92, in zhihu_process
    Xpath_wait_click_or_input(driver, '//div[@class="children"]//div[contains(@class, "withoutCollapse") and @title="z知乎"]')
  File "D:\00Code\0-old\auto_zhihu_clipper\utils\utils.py", line 25, in Xpath_wait_click_or_input
    wait_to_visible(driver, xpath)
  File "D:\00Code\0-old\auto_zhihu_clipper\utils\utils.py", line 14, in wait_to_visible
    ui.WebDriverWait(driver, timeout).until(EC.visibility_of_element_located((By.XPATH, Xpath)))
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 128, in __call__
    return _element_if_visible(_find_element(driver, self.locator))
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 415, in _find_element
    raise e
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 411, in _find_element
    return driver.find_element(*by)
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 978, in find_element
    'value': value})['value']
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\ngp\Anaconda3\envs\auto_zhihu_clipper\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: target frame detached
  (Session info: chrome=85.0.4183.121)
  (Driver info: chromedriver=72.0.3626.7 (efcef9a3ecda02b2132af215116a03852d08b9cb),platform=Windows NT 10.0.18363 x86_64)


